staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2313,6 +2315,9 @@ func describeJob(job *batchv1.Job, events *corev1.EventList) (string, error) {
 		if job.Spec.Parallelism != nil {
 			w.Write(LEVEL_0, "Parallelism:\t%d\n", *job.Spec.Parallelism)
 		}
+		if job.Spec.Suspend != nil {
```

Comment:
Can we also print:  `BackoffLimit` and `TTLSecondsAfterFinished`? These are other important GA-level fields. There is also `ManualSelector`, but I'm not sure it is relevant enough.

Reply:
My biggest concern is you're adding this between two important parameters, parallelism and completions, which I frequently look at together. I'd suggest to move suspend printing below completion mode or before parallelism, iow: 
your current output looks like:
```
Annotations:              <none>
Parallelism:              1
Suspend:                  false
Completions:              1
Completion Mode:          NonIndexed
Start Time:               Mon, 11 Dec 2023 15:18:09 +0100
```
but I'd suggest to have (suspend before parallelism):
```
Annotations:              <none>
Suspend:                  false
Parallelism:              1
Completions:              1
Completion Mode:          NonIndexed
Start Time:               Mon, 11 Dec 2023 15:18:09 +0100
```
or (suspend after completion mode) 
```
Annotations:              <none>
Parallelism:              1
Completions:              1
Completion Mode:          NonIndexed
Suspend:                  false
Start Time:               Mon, 11 Dec 2023 15:18:09 +0100
```

Reply:
@soltysh, any suggestion on where to place `BackoffLimit` and `TTLSecondsAfterFinished`? I'm assuming, going with your last suggestion, after `Suspend` and before `Start Time.` But I just wanted to check if you have an opinion here.

Reply:
For me pretty much every configuration works, just don't split the `Parallelism`, `completions` and `completionMode`, as pointed out. 

If I were to suggest something then maybe:
```
Annotations:      batch.kubernetes.io/cronjob-scheduled-timestamp: 2024-01-19T08:48:00Z
Controlled By:    CronJob/hello
Parallelism:      1
Completions:      1
Completion Mode:  NonIndexed
Backoff Limit:          6 
TTL Seconds After Finished: 21  # seems reasonable to be close to `Completed At`
Start Time:       Fri, 19 Jan 2024 09:48:00 +0100
Completed At:     Fri, 19 Jan 2024 09:48:13 +0100
Duration:         13s
```

Reply:
Done, it now looks like the following:

```
            Name:                        bar
            Namespace:                   foo
            Selector:                    
            Labels:                      <none>
            Annotations:                 <none>
            Completions:                 <unset>
            Suspend:                     true
            Backoff Limit:               1
            TTL Seconds After Finished:  123
            Pods Statuses:               0 Active / 0 Succeeded / 0 Failed
            Pod Template:
              Labels:  <none>
              Containers: <none>
              Volumes:         <none>
              Node-Selectors:  <none>
              Tolerations:     <none>
            Events:            <none>
```